nservicebus/azure-storage-queues/configuration.md

```diff
@@ -60,3 +60,10 @@ This feature can be enabled by specifying `.UseAccountNamesInsteadOfConnectionSt
 snippet:AzureStorageQueueUseAccountNamesInsteadOfConnectionStrings
 
 NOTE: This feature is not available in `NServiceBus.Azure.Transports.WindowsAzureStorageQueues` Versions 6 and below.
+
+## FIPS compliance
```

Comment:
I agree with @MarcinHoppe. No where do we say, FIPS or not. Starting here seems incorrect.